examples/monorepo/garn.ts

```diff
@@ -18,11 +18,9 @@ export const npmFrontend = garn.javascript.mkNpmProject({
   description: "frontend test app created by create-react-app",
   src: "frontend-npm",
   nodeVersion: "18",
-  testCommand: "npm run test -- --watchAll=false",
 });
 
 export const startAll = garn.processCompose({
   backend: backend.defaultExecutable!,
-  'yarn frontend': yarnFrontend.defaultExecutable!,
-  'npm frontend': npmFrontend.defaultExecutable!,
+  frontend: npmFrontend.devShell.shell`cd frontend-npm && npm install && npm start`,
```

Comment:
We don't run the yarn frontend here anymore?

Reply:
I figured maybe it was best to omit it for now until our yarn API becomes more stable. Happy to add it back if you feel differently though!

Reply:
I think you can do ```npmFrontend.shell`...```.

Reply:
And maybe replace the yarn project with the haskell one?